pagopa/io-app
# fix: [IOBP-332] Add correct pictograms to camera permission view in barcode scan screen

## Short description
This PR fixes the pictorgrams displayed in the camera permission requests view in the Barcode scan screen

| Not determined | Denied |
| --- | --- |
| <img src="https://github.com/pagopa/io-app/assets/6160324/2457c7d6-fadd-4b72-beea-36c1b9d12755" width="200" /> | <img src="https://github.com/pagopa/io-app/assets/6160324/e3decd8e-aead-49aa-8be3-a6e3273f6d54" width="200" /> |

## List of changes proposed in this pull request
- [ts/features/barcode/components/BarcodeScanBaseScreenComponent.tsx](https://github.com/pagopa/io-app/compare/IOBP-332-fix-barcode-scan-screen-camera-permission-pictograms?expand=1#diff-1e4e400edd0a8debdd16a6c728d260d849b977083ea5aa108a5ba43ce25ac4c2): added correct pictograms for permission request and permission denied views
- [ts/features/barcode/components/CameraPermissionView.tsx](https://github.com/pagopa/io-app/compare/IOBP-332-fix-barcode-scan-screen-camera-permission-pictograms?expand=1#diff-052d138722206aca70841ca9fe25e4ab0c8cc2266e26bc7e1b872dd97208e689): added `pictogram` in props and changed `pictogramStyle` to `light-content`.

## How to test
Navigate to the barcode scan screen and check that the pictograms are displayed correctly
